vscode/src/config.ts

```diff
@@ -27,8 +27,26 @@ export async function setTarget(target: TargetProfile) {
   );
 }
 
+export function getTargetFriendlyName(targetProfile?: string) {
+  switch (targetProfile) {
+    case "base":
+      return "Q#: QIR base";
+    case "unrestricted":
+      return "Q#: unrestricted";
+    default:
+      log.error("invalid target profile found");
+      return "Q#: invalid";
+  }
+}
+
 export function getEnableFormating(): boolean {
   return vscode.workspace
     .getConfiguration("Q#")
     .get<boolean>("enableFormatting", true);
 }
+
+export function getShowCircuitCodeLens(): boolean {
+  return vscode.workspace
+    .getConfiguration("Q#")
+    .get<boolean>("showCircuitCodeLens", false);
```

Comment:
Should we remove this now that we'll show it by default in dev builds? (Or at least default to 'true')

Reply:
I wanted to add a package name check for the "enable in dev builds" thing and doing that in a different PR. But if that sounds overly complicated I can just set it to true now.